### ISSUE TITLE
False positive for "The property X that you're trying to access does not exist in the class declaration." 

### ISSUE THREAD

**Issue by JohannesRudolph** [mgechev/codelyzer#273]
The following is perfectly legal angular template html: 

```
 <tr ngForm #newInterfaceForm="ngForm">
   <td class="td-edit">
     <button [disabled]="!newInterfaceForm.valid">Button</button>
   </td>
 </tr>
```

Note that this code is inside a table and only uses the ngForm directive, not the usual `<form>` tag.
Codelyzer 2.0.1 throws a false positive in this case: `...component.html[54, 149]: The property "newInterfaceForm" that you're trying to access does not exist in the class declaration.`

**Comment by JohannesRudolph** [comment]
I was able to fix it myself by adding a `.codelyzer.js` config as documented and adding the `[ngForm]` selector: 

```
module.exports = {

  // Custom predefined directives in case you get error for
  // missing property and you are using a template reference
  predefinedDirectives: [
    { selector: 'form, [ngForm]', exportAs: 'ngForm' },
   ...
  ],
}

```

You might want to consider adding this to the defaults as its an angular core directive (as opposed to the additional material design selectors that are in there too).

**Comment by mgechev** [comment]
This is fixed by codelyzer@3.0.0-beta.0.
